code-challenges.test.js

```diff
@@ -1,49 +1,172 @@
 // ASSESSMENT 4: JavaScript Coding Practical Questions with Jest
-
-// Please read all questions thoroughly
-// Pseudo coding is REQUIRED
-// If you get stuck, please leave comments to help us understand your thought process
-
-// Use test driven development to complete the following questions
-// Add appropriate dependencies: $ yarn add jest
-
-// Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
-
-
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
-
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
 const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+// // // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
 const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+// // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+
+describe("shuffleColors", () => {
+  it("takes in an array of colors, removes first item from array and shuffles the array of colors", () => {
+    const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+    const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+    expect(shuffleColors(colors1)).toEqual(expect.arrayHolding(["yellow", "blue", "pink", "green"]))
+    expect(shuffleColors(colors2)).toEqual(expect.arrayHolding(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]))
+     })
+})
```

Comment:
Test looks much better, just make sure you are using the correct matcher.  I would suggest taking a look at the link provided in the prompt to point you to the correct matcher.  I don't believe there is a jest matcher called arrayHolding.